### PR TITLE
Update release files

### PR DESCRIPTION
**Pull request by matentzn**
This PR is

1. An update of the sync files with the state of yesterday afternoon
2. A test for #420

### CODE REVIEW

**Review by matentzn — COMMENTED** [review]
This looks good to me (Approved).

Major changes in

- src/ontology/reports/omim_exclusion_reasons.robot.template.tsv
- src/ontology/reports/component_signature-omim.tsv

@sabrinatoro s curation seems reflected in

- src/ontology/reports/omim_mapping_status.tsv

@twhetzel @sabrinatoro I suggest you both block an hour to look at this diff not to find mistakes (I think there are not any), but to become more familiar with the kinds of files the sync pipeline creates.

**Comment by sabrinatoro** [comment]
@matentzn I don't see a new omim slurp file in this PR. I was expected to see one (I know there are some omim terms that should be coming in)

**Comment by sabrinatoro** [comment]
Note that the terms I was expecting in the slurp files are correctly reported in the src/ontology/reports/omim_mapping_status.tsv

**Comment by sabrinatoro** [comment]
Changes in src/ontology/reports/omim_exclusion_reasons.robot.template.tsv are (I didn't check all of them, but a few) records for genes. 
(this answer our question about whether we have a way to automatically exclude genes)

File: src/ontology/reports/doid_exclusion_reasons.robot.template.tsv [hunk 1]
```diff
@@ -530,6 +530,9 @@ DOID:0110638	MONDO:excludeNonDisease
 DOID:0110649	MONDO:excludeNonDisease
 DOID:0110748	MONDO:excludeNonDisease
 DOID:0110762	MONDO:excludeNonDisease
+DOID:0111201	MONDO:excludeNonDisease
+DOID:0111204	MONDO:excludeNonDisease
+DOID:0111208	MONDO:excludeNonDisease
```
Comment (sabrinatoro) [inline src/ontology/reports/doid_exclusion_reasons.robot.template.tsv:535]: @matentzn I don't know why these are in the exclusion list: there are currently as equivalent in Mondo

**Comment by sabrinatoro** [comment]
@twhetzel I am creating follow-up issues, but I think this can be merged. 
You can either merge it, or give me the ok to merge it please. Thanks

**Review by sabrinatoro — APPROVED** [review]

**Comment by twhetzel** [comment]
@sabrinatoro I'll leave it with your review as approved and will merge it.

**Pull request merged**